src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User user: users) {
```

Comment:
```suggestion
        for (User user : users) {
```
you can use IDEA autoformatter (Ctrl Alt L) when you're not sure

Reply:
maven didn't complain)